raiden-cli/src/routes/channels.ts

```diff
@@ -1,21 +1,33 @@
 import { Router, Request, Response, NextFunction } from 'express';
 import { first, pluck } from 'rxjs/operators';
-import { ErrorCodes, RaidenError, isntNil } from 'raiden-ts';
-import { Cli } from '../types';
+import { ErrorCodes, isntNil, ChannelState } from 'raiden-ts';
+import { Cli, ApiChannelState } from '../types';
 import {
   isInvalidParameterError,
   isTransactionWouldFailError,
   validateOptionalAddressParameter,
 } from '../utils/validation';
 import { flattenChannelDictionary, transformSdkChannelFormatToApi } from '../utils/channels';
 
-function isConflictError(error: RaidenError): boolean {
+function isConflictError(error: Error): boolean {
   return (
     [ErrorCodes.RDN_UNKNOWN_TOKEN_NETWORK, ErrorCodes.CNL_INVALID_STATE].includes(error.message) ||
     isTransactionWouldFailError(error)
   );
 }
 
+function isInsuficientFundsError(error: { message: string; code?: string | number }): boolean {
+  return (
+    error.code === 'INSUFFICIENT_FUNDS' ||
```

Comment:
Is this thrown by EthersJS? :thinking:

Reply:
I don't know where it came from, it was already there, I just kept it. EthersJS error code's usually are numeric, afaik